src/main/java/com/synopsys/integration/blackduck/imageinspector/api/name/ImageNameResolver.java

```diff
@@ -7,20 +7,29 @@
  */
 package com.synopsys.integration.blackduck.imageinspector.api.name;
 
-import java.util.Optional;
-
 import com.synopsys.integration.blackduck.imageinspector.image.common.RepoTag;
 import org.apache.commons.lang3.StringUtils;
 import org.jetbrains.annotations.Nullable;
 import org.springframework.stereotype.Component;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 @Component
 public class ImageNameResolver {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
     public RepoTag resolve(@Nullable String foundImageName, @Nullable String givenRepo, @Nullable String givenTag) {
         if (StringUtils.isBlank(foundImageName)) {
             return new RepoTag(givenRepo, givenTag);
         }
+        if (!foundImageName.contains(":")) {
+            if (givenTag.isBlank()) {
+                givenTag = "latest";
+            }
+            logger.trace(String.format("foundImageName %s in the manifest is not in repo:tag format, hence resolver will not use foundImageName for name resolution. Resolving RepoTag with givenRepo as %s and givenTag as %s", foundImageName, givenRepo, givenTag));
+            return new RepoTag(givenRepo, givenTag);
+        }
```

Comment:
Handles cases where the OCI image reference name is not in the repo:tag format. 2 possible scenarios exist here:
1) If the input is a tar file, as per our docs we recommend that the user provide us with the image name and image tag for correct project/version name resolution using the `detect.docker.passthrough.docker.image.repo` and `detect.docker.passthrough.docker.image.tag` properties.  If the tag is not provided, the project version is resolved as default tag value i.e.  `latest`. If the repo name is not provided, the file name becomes the project name.

2) If the input is not a tar file, givenRepo and givenTag will always have resolved values from upstream code.